vllm/envs.py

```diff
@@ -205,7 +206,18 @@ def get_default_config_root():
         os.environ.get("VLLM_TEST_DYNAMO_FULLGRAPH_CAPTURE", "1") != "0"),
     "VLLM_TORCH_COMPILE_LEVEL":
     lambda: int(os.environ.get("VLLM_TORCH_COMPILE_LEVEL", "0")),
-
+    # Fine-grained control over which custom ops to enable.
+    # Use 'all' to enable all, 'none' to disable all.
+    # Also specify a list of custom op names to enable,
+    # or disable if the name is prefixed with a '-'.
+    # Examples:
+    # - 'all,-op1' to enable all except op1
+    # - 'none,op1,op2' to enable only op1 and op2
+    # By default, all custom ops are enabled when running without Inductor
+    # and disabled when running with Inductor (compile_level >= Inductor).
+    "VLLM_ENABLE_CUSTOM_OPS":
```

Comment:
we can rename it to `VLLM_CUSTOM_OPS` , with the following semantic:

if `envs.VLLM_TORCH_COMPILE_LEVEL >= CompilationLevel.INDUCTOR` , all custom ops are disabled by default, and we only check `+rms_norm` etc to enable some of them.

if `envs.VLLM_TORCH_COMPILE_LEVEL < CompilationLevel.INDUCTOR` , all custom ops are enabled by default, and we only check `-rms_norm` etc to disable some of them.

Reply:
I assume we still want to support `all` or `none` though if someone wants to disable all custom ops without inductor or enable all with inductor?